muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/workspace/WorkspaceJournalBackingBean.java

```diff
@@ -209,6 +215,16 @@ public Long getStudentId() {
     return studentId;
   }
   
+  public Long getPage() {
+    if (page == null)
```

Comment:
Could also be just `return page == null ? 1l : page;` but it's admittedly a tomato/potato kind of thing.